Sources/MapboxMaps/Style/Style.swift

```diff
@@ -427,7 +427,14 @@ public struct StyleTransition: Codable {
     ///   - duration: Time for transition in milliseconds.
     ///   - delay: Time before transition begins in milliseconds.
     public init(duration: TimeInterval, delay: TimeInterval) {
-        self.duration = duration
-        self.delay = delay
+        self.duration = duration * 1000
+        self.delay = delay * 1000
```

Comment:
`self.duration` and `self.delay` are also of type `TimeInterval`, so we shouldn't do the conversion here. If the goal is for the JSON serialized representation of a `StyleTransition` to represent these time intervals in ms, you can handle the conversions in custom `encode(to:)` and `init(from:)` implementations. Your goal is for the JSON to use ms, and this struct to use `TimeInterval`.